arraycontext/container/arithmetic.py

```diff
@@ -149,22 +296,16 @@ def with_container_arithmetic(
 
     :arg bcast_number: If *True*, numbers broadcast over the container
         (with the container as the 'outer' structure).
-    :arg _bcast_actx_array_type: If *True*, instances of base array types of the
```

Comment:
Yeah, that wasn't a lot of code to begin with, so shouldn't be a big issue. I recall mostly wanting to ax it because `force_device_scalar=False` was getting deprecated, not because of broadcasting.